### DISCUSSION TITLE
Weird nodes connection

### DISCUSSION THREAD

**Discussion by Tofandel** [Chia-Network/chia-blockchain#5995] [Support]
Is this normal?
![image](https://user-images.githubusercontent.com/6115458/119484932-cda65800-bd56-11eb-828f-aa76b3aa6510.png)

![image](https://user-images.githubusercontent.com/6115458/119484887-c0896900-bd56-11eb-89fa-fd08f4387479.png)

I have a ton of those FULL_NODE + WALLET 172.17.0.1 connecting to my full node, (I have only one local wallet and it's connected via 127.0.0.1) so they don't seem to be my wallets (which is weird since 172.17.0.1 is a private IP address) and they are crapping out the sync, my node is not syncing while they are connected..

If I remove them, they reconnect within a minute

Is it some kind of exploit or just a bug? Or my local wallet will connect multiple times to sync and totally normal?

**Comment by tiny0little** [comment]
is 172.17.0.1 your router? it translates SRC IP to itself. it does SNAT as well as DNAT.
routers with multiple WAN interfaces usually do that {or firmware which is suitable for multiple WAN interfaces}

all these connections are actual NODEs on the internet, they prove that your inbound `tcp8444` works

on another note, connections to `tcp8449` are wallet connections, I think you should not see them from internet, unless you have multiple LANs off your router

**Reply by Tofandel** [reply]
Mmm not my router but probably the docker container on which it's running uses this as a local network interface

Does that mean those wallet connections are abnormal?

**Reply by tiny0little** [reply]
I also have them

```
$ chia show -c| grep WAL
WALLET    192.168.0.1                            62727/8449  2429ee36... May 25 14:59:51     13.6|0.0    
WALLET    192.168.0.1                            56058/8449  de83b84f... May 25 14:59:52      2.4|0.0    
WALLET    127.0.0.1                              59038/8449  d5f77347... May 25 14:59:51     34.1|0.0 
```

**Reply by Tofandel** [reply]
Ok I'm finally synced after a lot of messing around, the connections are still there and seem indeed normal behavior

**Reply by lvpcguru** [reply]
Nice to know!

**Comment by lvpcguru** [comment]
Also you dont have port 8449 open to the internet do you?

**Reply by Tofandel** [reply]
No I don't, I do have a firewall only allowing 8444

**Reply by lvpcguru** [reply]
ohh your using docker. that could change several things in the connections. Sorry havnt played with the docker deployment so i dont know if its normal in your case.

**Comment by lvpcguru** [comment]
You should make sure you have a firewall setup to allow 8444 only in from the internet and the remaining chia ports and administration from your local network.

**Comment by BrandtH22** [comment]
@Tofandel , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
